transaction/summary/src/verifier.rs

```diff
@@ -155,7 +84,7 @@ impl TxSummaryStreamingVerifier {
     /// Returns:
     /// * A properly initialized TxSummaryStreamingVerifier
     pub fn new(
-        extended_message_digest: &[u8; 32],
+        extended_message_digest: &[u8],
```

Comment:
is there a good reason for this change? i thought we reviewed this earlier and reverted it

Reply:
ahhh we did?! i must have done _something_ wrong when restacking :-/